realm-transformer/build.gradle

```diff
@@ -25,12 +25,23 @@ repositories {
     jcenter()
 }
 
+configurations {
+    provided
+    compile.extendsFrom provided
+}
+
+sourceSets {
+    main {
+        compileClasspath += configurations.provided
+    }
+}
+
 dependencies {
     compile localGroovy()
     compile gradleApi()
     compile "io.realm:realm-annotations:${version}"
     compile 'com.android.tools.build:gradle:1.5.0'
-    compile 'com.android.tools.build:transform-api:1.5.0'
+    provided 'com.android.tools.build:transform-api:1.5.0'
```

Comment:
Isn't the transform API a dependency for the build tools already? Shouldn't we be able to just remove this? This would also mean it should be the build tools that should be a provided dependency?